### ISSUE TITLE
Is 2.1.0 an official release? If so why is it tagged as 2.1.0rc2? Also, "pip install apache-superset==2.1.0" fails.

### ISSUE THREAD

**Issue by dat-linux** [apache/superset#23364]
Release metadata on PyPi (https://pypi.org/rss/project/apache-superset/releases.xml) shows up as "2.1.0", ie a minor release:
```
<item>
      <title>2.1.0</title>
      <link>https://pypi.org/project/apache-superset/2.1.0/</link>
      <description>A modern, enterprise-ready business intelligence web application</description>
      <author>dev@superset.apache.org</author>
      <pubDate>Tue, 14 Mar 2023 02:00:56 GMT</pubDate>
    </item>
```

However installing either as `apache-superset==2.1.0` or `apache-superset==2.1.0rc2` fails:

```
pip install apache-superset==2.1.0                                                                                                                                                  
ERROR: Could not find a version that satisfies the requirement apache-superset==2.1.0 (from versions: 0.34.0, 0.34.1, 0.35.1, 0.35.2, 0.36.0, 0.37.0, 0.37.1, 0.37.2, 0.38.0, 0.38.1, 1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.4.0, 1.4.1, 1.4.2, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 2.0.0, 2.0.1, 2.1.0rc2)                                                                                                                                                                 
ERROR: No matching distribution found for apache-superset==2.1.0   
                                                                                                                                                                     
pip install apache-superset==2.1.0rc2                                                                                                                                               
Collecting apache-superset==2.1.0rc2
  Downloading apache-superset-2.1.0rc2.tar.gz (46.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 46.9/46.9 MB 3.7 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
  WARNING: Requested apache-superset==2.1.0rc2 from https://files.pythonhosted.org/packages/6a/70/791ade4ac0a8b49d77b7110de6ad69ccea3f4ff38f51cb16f032ac6fa158/apache-superset-2.1.0rc2.tar.gz, but installing version 2.1.0
Discarding https://files.pythonhosted.org/packages/6a/70/791ade4ac0a8b49d77b7110de6ad69ccea3f4ff38f51cb16f032ac6fa158/apache-superset-2.1.0rc2.tar.gz (from https://pypi.org/simple/apache-superset/) (requires-python:~=3.8): Requested apache-superset==2.1.0rc2 from https://files.pythonhosted.org/packages/6a/70/791ade4ac0a8b49d77b7110de6ad69ccea3f4ff38f51cb16f032ac6fa158/apache-superset-2.1.0rc2.tar.gz has inconsistent version: expected '2.1.0rc2', but metadata has '2.1.0'                                                                                                                                                                                                                              
ERROR: Could not find a version that satisfies the requirement apache-superset==2.1.0rc2 (from versions: 0.34.0, 0.34.1, 0.35.1, 0.35.2, 0.36.0, 0.37.0, 0.37.1, 0.37.2, 0.38.0, 0.38.1, 1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.4.0, 1.4.1, 1.4.2, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 2.0.0, 2.0.1, 2.1.0rc2)                                                                                                                                                              
ERROR: No matching distribution found for apache-superset==2.1.0rc2 
```

And when using `pip install apache-superset` it tries to install 2.0.1 instead:

```
pip install apache-superset
Collecting apache-superset
  Downloading apache-superset-2.0.1.tar.gz (42.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 42.5/42.5 MB 3.8 MB/s eta 0:00:00
```
The PyPi page for Superset - https://pypi.org/project/apache-superset/ -  identifies 2.1.0 as the latest version, and the tag below identifies the release as v2.1.0rc, but the pip version as 2.1.0.

**Comment by sfirke** [comment]
the "rc" means Release Candidate.  2.1.0rc2, the second release candidate, is now being voted on by the community.  rc1 was voted down.  When a release candidate is approved, it will be released as the official 2.1.0.  Currently 2.0.1 is the latest official release.

**Comment by dat-linux** [comment]
> the "rc" means Release Candidate. 2.1.0rc2, the second release candidate, is now being voted on by the community. rc1 was voted down. When a release candidate is approved, it will be released as the official 2.1.0. Currently 2.0.1 is the latest official release.

OK, but why is 2.1.0 the latest release ID on PyPi, but not installable?

**Comment by asmaier** [comment]
To install release candidates with pip use `--pre` like

`pip install --pre apache-superset`

see https://jugmac00.github.io/til/how-to-install-release-candidates/

**Comment by dat-linux** [comment]
> To install release candidates with pip use `--pre` like
> 
> `pip install --pre apache-superset`
> 
> see https://jugmac00.github.io/til/how-to-install-release-candidates/

Does the same thing, v 2.0.1. The problem is that pip has the latest stable version as 2.1.0, but no such release is available.  There is **No matching distribution found for apache-superset==2.1.0** or  **apache-superset==2.1.0rc2**

**Comment by eschutho** [comment]
Yes, as you mentioned this issue should have been fixed as of 2.1.0rc3. I'll close this issue, but feel free to reopen if you still see any inconsistencies.
